pytest_fixtures/component/compute_resource.py

```diff
@@ -0,0 +1,30 @@
+"""
+Fixtures for Compute Resource(s).
+"""
+
+from fauxfactory import gen_string
+import pytest
+
+from robottelo.config import settings
+from robottelo.constants import FOREMAN_PROVIDERS, LIBVIRT_RESOURCE_URL
+
+LIBVIRT_URL = LIBVIRT_RESOURCE_URL % settings.libvirt.libvirt_hostname
+
+
+@pytest.fixture(scope='module')
+def module_libvirt_cr(module_target_sat, module_org, module_location):
```

Comment:
Is it possible to use existing `module_cr_libvirt` fixture for this case or it can be updated for your use case, instead of writing a new fixture?

Reply:
Oh, nice, didn't find this fixture, I can definitely use it as a "base" for this one.

Reply:
Actually for the test, a "dummy" compute resource to fill the form fields will suffice.
So I will go with the existing `module_cr_libvirt` fixture and I will drop this one.